_src/css/helpcenter/_search-bar.scss

```diff
@@ -0,0 +1,27 @@
+#docsSearch {
+  background-color: $csl-blue;
+  color: white;
+
+  // Insert the image above the search box
+  h1 {
+    margin-bottom: 0;
+
+    &:after {
+      content: ' ';
+      background-image: url('images/helpcenter/homepage.svg') center no-repeat;
+      display: block;
+      height: 282px;
+      margin-top: 15px;
```

Comment:
Do these work correctly on mobile?

Reply:
Good question! It looks like we should scale down the image a bit on mobile.